src/test/java/com/rackspacecloud/metrics/ingestionservice/RollupIngestionServiceTests.java

```diff
@@ -98,19 +93,18 @@ public void test_Rollup1440M_Consume() throws Exception {
 
     private void testForGivenTopic(String topicToSendData) throws Exception {
         // Mock influxDB ingestion call
-//        when(this.influxDBHelperMock.ingestToInfluxDb(anyString(), anyString(), anyString(), anyString()))
-//                .thenReturn(true);
-//
-//        for(int i = 0; i < 1; i++) {
-//            sender.sendRollup(
-//                    MockMetricRollupHelper.getValidMetricRollup(i, "hybrid:1667601", true),
-//                    topicToSendData);
-//        }
-//
-//        Thread.sleep(10*1000L); // wait for a few sec for consumer to process some records
-//
-//        long batchProcessed = rollupListener.getBatchProcessedCount();
-//        Assert.assertTrue(
-//                String.format("Failed with batchProcessed count [%s]", batchProcessed), batchProcessed > 0);
+        doNothing().when(this.influxDBHelperMock).ingestToInfluxDb(anyString(), anyString(), anyString(), anyString());
+
+        for(int i = 0; i < 1; i++) {
+            sender.sendRollup(
+                    MockMetricRollupHelper.getValidMetricRollup(i, "hybrid:1667601", true),
+                    topicToSendData);
+        }
+
+        Thread.sleep(10*1000L); // wait for a few sec for consumer to process some records
+
+        long batchProcessed = rollupListener.getBatchProcessedCount();
+        Assert.assertTrue(
```

Comment:
It would be even better if you had a Mockito `verify` of `influxDBHelperMock.ingestToInfluxDb`, but I could see the setup for that verification being tedious. Also, FYI you can use this

https://static.javadoc.io/org.mockito/mockito-core/2.27.0/org/mockito/Mockito.html#22

to verify yet still allow some amount of duration to pass for asynchronous processing.